packages/agoric-cli/src/lib/format.js

```diff
@@ -30,16 +30,18 @@ const exampleAsset = {
   issuer: makeBoardRemote({ boardId: null, iface: undefined }),
   petname: 'Agoric staking token',
 };
-/** @typedef {import('@agoric/vats/tools/board-utils.js').VBankAssetDetail & { brand: import('@agoric/vats/tools/board-utils.js').BoardRemote }} AssetDescriptor */
+/** @typedef {import('@agoric/vats/tools/board-utils.js').VBankAssetDetail } AssetDescriptor */
 
-/** @param {AssetDescriptor[]} assets */
+/**
+ * @param {AssetDescriptor[]} assets
+ * @returns {(a: Amount) => [string, number | any[]]}
+ */
 export const makeAmountFormatter = assets => amt => {
-  const {
-    brand: { boardId },
-    value,
-  } = amt;
+  const { brand, value } = amt;
+  // @ts-expect-error XXX BoardRemote
```

Comment:
why not type AssetDescriptor?

Reply:
I don't understand. `AssetDescriptor` is used on like 36.